src/theming.tsx

```diff
@@ -271,135 +280,185 @@ export const buildTheme = (
     ? LIGHT_MODE_HIGH_CONTRAST_COLOURS
     : LIGHT_MODE_COLOURS;
 
-  const overrides = {
+  const componentOverrides = {
+    MuiPaper: {
+      styleOverrides: { root: { backgroundImage: 'unset' } },
+    },
     MuiLink: {
-      root: {
-        color: colours.blue,
+      defaultProps: {
+        underline: 'hover' as 'always' | 'none' | 'hover' | undefined,
       },
-      button: {
-        fontFamily: `"Roboto", "Helvetica", "Arial", sans-serif`,
-        verticalAlign: 'none',
+      styleOverrides: {
+        root: {
+          color: colours.blue,
+        },
       },
     },
     MuiTabs: {
-      indicator: {
-        color: STATIC_COLOURS.darkBlue,
-        textDecoration: 'underline',
+      styleOverrides: {
+        indicator: {
+          color: STATIC_COLOURS.darkBlue,
+          textDecoration: 'underline',
+        },
       },
     },
-    MuiFormLabel: {
-      root: {
-        '&$error': {
-          color: colours.red,
+    MuiTab: {
+      styleOverrides: {
+        textColorSecondary: {
+          [`&.${tabClasses.selected}`]: {
+            color: darkModePreference ? '#FFFFFF' : colours.blue,
+          },
         },
-        '&$focused': {
-          color: colours.blue,
+      },
+    },
+    MuiCheckbox: {
+      styleOverrides: {
+        colorPrimary: {
+          [`&.${checkboxClasses.checked}`]: {
+            color: colours.blue,
+          },
+          [`&.${checkboxClasses.indeterminate}`]: {
+            color: colours.grey,
+          },
         },
       },
-      asterisk: {
-        '&$error': {
-          color: colours.red,
+    },
+    MuiFormLabel: {
+      styleOverrides: {
+        root: {
+          '&$error': {
```

Comment:
Shouldn't this line match the one below it? Since I thought the `$` notation doesn't work anymore